cmd/context/utils.go

```diff
@@ -222,9 +222,12 @@ func LoadContextFromPath(ctx context.Context, namespace, k8sContext, path string
 	}
 	ctxResource, err := model.GetContextResource(path)
 	if err != nil {
-		if !errors.Is(err, oktetoErrors.ErrManifestNotFound) {
+		// model.GetContextResource will only return discovery.ErrOktetoManifestNotFound in the event that no manifest is found
+		if !errors.Is(err, discovery.ErrOktetoManifestNotFound) {
 			return err
 		}
+		// here ctxResource is nil -> so we get a panic when the func
+		ctxResource = &model.ContextResource{}
```

Comment:
@ifbyol when reverting the GetContextResource, now i get the no manifest found error, which is the exception, then ctxResource is nil, and i get a nil pointer at line 233 (inside UpdateNamespace) because ctxResource is nil :/ is this a good fix? i dont feel like  it.. i thought that maybe in the case that no manifest found, GetContextResource will return empty ctxResource?

Reply:
I think it should behave the same way that the deploy, it is almost the same thing. I'll try to get some time to take a look to this code carefully and understand why it fails

Reply:
I've gone around this.... updated the PR, honestly i think this should return err in case of err... is "loading the context from manifest" if no manifest, no context info, so we init with the defaults or flag values as we do in the deploy and now destroy...
Updated code